modules/statics/src/coins.ts

```diff
@@ -1465,6 +1465,14 @@ export const coins = CoinMap.fromCoins([
     '0x1a4b46696b2bb4794eb3d4c26f1c55f9170fa4c5',
     UnderlyingAsset.BIT
   ),
+  erc20(
+    '83214bf2-eec7-4225-b398-41e065e23311',
+    'blur',
+    'Blur',
+    18,
+    '0x5283D291DBCF85356A21bA090E6db59121208b44',
```

Comment:
All contract addresses have to be in lowerCase

Reply:
@sachushaji updated the contract address to lower case